dbt/scripts/utils/helpers.py

```diff
@@ -45,10 +56,18 @@ def create_logger(
             log_group_name=log_group_name,
             stream_name=stream_name,
         )
+
+        # Don't push timestamps to CloudWatch, since CloudWatch adds its own
+        # timestamps
+        cw_handler.setFormatter(
+            logging.Formatter(fmt="%(levelname)s - %(message)s")
+        )
```

Comment:
Previously we had not been formatting the logs that we push to CloudWatch (or to the local file), which meant the log level name (like `INFO` or `ERROR) was not being included in the CloudWatch logs. This meant that we had to add the string `Error` manually during our error handling in order to trigger our metrics filter alarm. By adding this formatter to the CloudWatch log handler, we can ensure we always push the log level name, and avoid having to add a custom string to trigger the metrics filter alarm.

Note that we define a custom formatter here and skip the timestamp that we use in the stream and file handlers because CloudWatch adds its own timestamps to log lines.